_includes/pbc/all/install-features/202304.0/install-push-notification-feature.md

```diff
@@ -0,0 +1,698 @@
+
+    /**
+     * @return array<int, \Spryker\Zed\PushNotificationExtension\Dependency\Plugin\PushNotificationSenderPluginInterface>
+     */
+    protected function getPushNotificationSenderPlugins(): array
+    {
+        return [
+            new PushNotificationWebPushPhpPushNotificationSenderPlugin(),
+        ];
+    }
+}
+
+```
+</details>
+
+2. Enable the following installer plugins:
+
+| PLUGIN                                            | SPECIFICATION                                                                      | PREREQUISITES | NAMESPACE                                                             |
+|---------------------------------------------------|------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------|
+| PushNotificationWebPushPhpProviderInstallerPlugin | Adds the `web-push` provider to the list of available push notification providers. |               | Spryker\Zed\PushNotificationWebPushPhp\Communication\Plugin\Installer |
+
+**src/Pyz/Zed/Installer/InstallerDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+use Spryker\Zed\PushNotificationWebPushPhp\Communication\Plugin\Installer\PushNotificationWebPushPhpProviderInstallerPlugin;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface|\Spryker\Zed\InstallerExtension\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins(): array
+    {   
+        return [
+            new PushNotificationWebPushPhpProviderInstallerPlugin(),
+        ];
+    }
+}
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the installer plugin works correctly:
+
+1. Execute install plugins: 
+```bash
+docker/sdk console setup:init-db`
+```
+
+2. Check that the `web-push-php` push notification provider exists in the `spy_push_notification_provider` database table.
+
+{% endinfo_block %}
+
+3. Enable the following console command plugins:
+
+| PLUGIN                                           | SPECIFICATION                                                    | PREREQUISITES | NAMESPACE                                          |
+|--------------------------------------------------|------------------------------------------------------------------|---------------|----------------------------------------------------|
+| SendPushNotificationConsole                      | Sends notifications in an async way.                             |               | Spryker\Zed\PushNotification\Communication\Console |
+| DeleteExpiredPushNotificationSubscriptionConsole | Delete expired push notification subscriptions from Persistence. |               | Spryker\Zed\PushNotification\Communication\Console |
```

Comment:
If its'a a layer, entity, folder, etc name, then enclose it in `code-fond` as follows:
```suggestion
| DeleteExpiredPushNotificationSubscriptionConsole | Delete expired push notification subscriptions from `Persistence`. |               | Spryker\Zed\PushNotification\Communication\Console |
```